src/components/books/BookItem.vue

```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+a(href='#')
+  .book-item(:class='{ "book-item-small": small }')
+    img.book-item-image(alt='book image' src='' v-bind:src='book.image_url || "assets/book_logo.png"')
+    p.book-item-title(v-if='!small')
+      | {{book.title}}
+    p.book-item-author(v-if='!small')
+      | {{book.author}}
+</template>
+<script>
+export default {
+  props: {
+    book: {
+      type: Object,
+      required: false,
+      default: undefined
+    }
+  },
+  data() {
+    return {
+      small: null
```

Comment:
false instead?

Reply:
If you are not going to change the data from a function, you can move this small to a prop instead i think 🤔